src/interactive/azext_interactive/azclishell/configuration.py

```diff
@@ -41,8 +45,7 @@ def help_text(values):
     """ reformats the help text """
     result = ""
     for key in values:
-        result += key + ' '.join('' for x in range(GESTURE_LENGTH - len(key))) +\
-                        ': ' + values[key] + '\n'
+        result += key + ' '.join('' for x in range(GESTURE_LENGTH - len(key))) + ': ' + values[key] + '\n'
```

Comment:
Maybe it is necessary to add some comments for the string joining, like adding an example.